gc/base/MemoryManager.cpp

```diff
@@ -569,6 +569,7 @@ MM_MemoryManager::doubleMapArraylet(MM_MemoryHandle* handle, MM_EnvironmentBase
 	Assert_MM_true(NULL != handle);
 	MM_VirtualMemory* memory = handle->getVirtualMemory();
 	Assert_MM_true(NULL != memory);
+
```

Comment:
this is non-functional change so this file should not be touched at all and excluded from change set

Reply:
Yes, slipped away. Fixing it.